src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{

```

Comment:
There should be a space after `Machine` to follow proper Java coding conventions.